libs/shared/barista-definitions/src/lib/barista-definitions.ts

```diff
@@ -155,3 +156,13 @@ export type BaContentTypes =
   | BaIndexPageContent
   | BaIconOverviewPageContent
   | BaCategoryNavigationContent;
+
+export interface TOC {
```

Comment:
In barista we prefix everything with `Ba` please add it in the sake of consistency.
And please stop using abbreviation as we agreed on this topic in our [coding standards](https://github.com/dynatrace-oss/barista/blob/master/CODING_STANDARDS.md#naming)

If this object is an array of Headlines please explain to me why it is needed – then the property can be of type `BaTableOfContentsHeadline[]` instead of `TOC`

```suggestion
export interface BaTableOfContents {
```